polkadot-protocol/protocol-operations/offchain-operations.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Off-Chain Operations
+description: Overview of Off-Chain workers in the Polkadot SDK, including their structure and the basic operations that can be performed on them.
+---
+
+# Off-Chain Operations
+
+## Introduction
+
+There are many use cases where you might want to query data from an Off-Chain source or process data without using On-Chain resources before updating the on-chain state. The conventional way of incorporating Off-Chain data involves connecting to oracles to supply the data from some traditional source. Although using oracles is one approach to working with Off-Chain data sources, there are limitations to the security, scalability, and infrastructure efficiency that oracles can provide.
+
+To make the Off-Chain data integration more secure and efficient, the Polkadot SDK supports Off-Chain operations through the following features:
+
+- Off-Chain workers are a subsystem of components that enable the execution of long-running and possibly non-deterministic tasks, such as:
```

Comment:
It will be clearer if you put titles to each of these "Parent" List items. Something like:
-  long-running and possibly non-deterministic tasks
- off-chain storage
- off-chain indexing

It is quite challenging to read as it. You can even link the sections below if you can make it match